components/blocks/ResendConfirmationBar/ResendConfirmationBar.vue

```diff
@@ -28,8 +27,40 @@ async function resend() {
 </script>
 
 <template>
-  <div>
-    <button @click="resend">Confirm</button>
-    <p v-if="showErrorText" class="text-red-600">{{ errorText }}</p>
+  <div class="resend-confirmation-bar__container">
+    <div class="resend-confirmation-bar__content">
+      <div class="resend-confirmation-bar__text">
+        <i-svg-circle-info />
+        <span>{{ $t('needToConfirm') }}</span>
+      </div>
+      <BaseButton class="resend-confirmation__button" @click="resend">
+        {{ $t('resendConfirmation') }}
+      </BaseButton>
+    </div>
+    <p v-if="showErrorText" class="error-text">{{ errorText }}</p>
```

Comment:
Man this raises a dilemma; how do we translate error messages in the future. I've never had to deal with that.

Reply:
I imagine we can just have it run through our `i18n` system, but I'd have to look into that more